apps/explorer/src/components/footer/Footer.tsx

```diff
@@ -2,97 +2,64 @@
+                            <Text color="white" variant="p4/semibold">
+                                All rights reserved
+                            </Text>
+                        </div>
                     </div>
+                    <FooterLinks links={footerLinks} />
                 </div>
+            </nav>
+        </footer>
+    );
+}
 
-                <div>
-                    <h6>Read</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://medium.com/mysten-labs">
-                                Blog
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://github.com/MystenLabs/sui/blob/main/doc/paper/sui.pdf">
-                                Whitepaper
-                            </Link>
-                        </li>
-                    </ul>
-                </div>
-                <div>
-                    <h6>Build</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://docs.sui.io/">Docs</Link>
-                        </li>
-                        <li>
-                            <Link href="https://github.com/MystenLabs">
-                                GitHub
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://discord.gg/sui">Discord</Link>
-                        </li>
-                    </ul>
-                </div>
-                <div>
-                    <h6>Follow</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://mystenlabs.com/#community">
-                                Press
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://twitter.com/SuiNetwork">
-                                Twitter
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://www.linkedin.com/company/mysten-labs/">
-                                LinkedIn
-                            </Link>
-                        </li>
-                    </ul>
-                </div>
-                <div>
-                    <h6>Legal</h6>
-                    <ul>
-                        <li>
-                            <Link href="https://mystenlabs.com/legal?content=terms">
-                                Terms & Conditions
-                            </Link>
-                        </li>
-                        <li>
-                            <Link href="https://mystenlabs.com/legal?content=privacy">
-                                Privacy Policy
-                            </Link>
-                        </li>
+function FooterLinks({ links }: { links: FooterItems }) {
```

Comment:
I'd put this above footer, so that usage order is consistent (i.e. this is defined before it is is used in the Footer component).

I also think if you want you could carve this out into a separate file but not essential.